includes/fraud-prevention/class-buyer-fingerprinting-service.php

```diff
@@ -0,0 +1,87 @@
+<?php
+/**
+ * Class Buyer_Fingerprinting_Service
+ *
+ * @package WCPay\Fraud_Prevention
+ */
+
+namespace WCPay\Fraud_Prevention;
+
+/**
+ * Class Buyer_Fingerprinting_Service
+ */
+class Buyer_Fingerprinting_Service {
+	/**
+	 * Singleton instance.
+	 *
+	 * @var Buyer_Fingerprinting_Service
+	 */
+	private static $instance;
+
+	/**
+	 * Fraud prevention service instance.
+	 *
+	 * @var Fraud_Prevention_Service
+	 */
+	private $fraud_prevention_service;
```

Comment:
What is the reason for having `Fraud_Prevention_Service` here? I don't see any usage.